tasks/main.yml

```diff
@@ -122,6 +131,26 @@
   with_items:
     - pacemaker
     - corosync
+  when: not cluster_is_remote | bool
+
+- name: Get cluster status
+  command: pcs cluster status
```

Comment:
- [ ] this works only on CentOS 8.x where `pcs cluster status` contains list of remote nodes, however on CentOS 7.8 and 6.10 this is not case as those are visible only in `pcs status`. Using `pcs status` would make more sense here to make this usable also in CentOS 7.8.
~~~
=== CentOS 8.2
# pcs cluster status
Cluster Status:
 Cluster Summary:
   * Stack: corosync
   * Current DC: fastvm-centos-8-2-54 (version 2.0.3-5.el8_2.1-4b1f869f0f) - partition with quorum
   * Last updated: Sun Aug  9 22:36:44 2020
   * Last change:  Sun Aug  9 22:34:12 2020 by hacluster via crmd on fastvm-centos-8-2-54
   * 3 nodes configured
   * 4 resource instances configured
 Node List:
   * Online: [ fastvm-centos-8-2-54 fastvm-centos-8-2-55 ]
   * RemoteOnline: [ fastvm-centos-8-2-56 ]

PCSD Status:
  fastvm-centos-8-2-54: Online
  fastvm-centos-8-2-55: Online
[root@fastvm-centos-8-2-54 ~]# pcs cluster status
Cluster Status:
 Cluster Summary:
   * Stack: corosync
   * Current DC: fastvm-centos-8-2-54 (version 2.0.3-5.el8_2.1-4b1f869f0f) - partition with quorum
   * Last updated: Sun Aug  9 22:38:44 2020
   * Last change:  Sun Aug  9 22:34:12 2020 by hacluster via crmd on fastvm-centos-8-2-54
   * 3 nodes configured
   * 4 resource instances configured
 Node List:
   * Online: [ fastvm-centos-8-2-54 fastvm-centos-8-2-55 ]
   * RemoteOnline: [ fastvm-centos-8-2-56 ]

PCSD Status:
  fastvm-centos-8-2-54: Online
  fastvm-centos-8-2-55: Online
~~~

~~~
=== CentOS 7.8
# pcs cluster status
Cluster Status:
 Stack: corosync
 Current DC: fastvm-centos-7-8-51 (version 1.1.21-4.el7-f14e36fd43) - partition with quorum
 Last updated: Sun Aug  9 22:38:49 2020
 Last change: Sun Aug  9 22:22:25 2020 by root via cibadmin on fastvm-centos-7-8-51
 3 nodes configured
 4 resources configured

PCSD Status:
  fastvm-centos-7-8-52: Online
  fastvm-centos-7-8-51: Online
~~~

Reply:
Thanks, I wasn't aware of that difference.